bears/general/QuotesBear.py

```diff
@@ -34,9 +35,16 @@ def correct_single_line_str(self, filename, file, sourcerange,
         str_contents = file[sourcerange.start.line - 1][
                        sourcerange.start.column:sourcerange.end.column-1]
 
-        if preferred_quotation in str_contents:
+        if (preferred_quotation in str_contents and
+                not force_preferred_quotation):
             return
 
+        """
+        Escape the preffered quotes if its already present.
+        """
+        add_escape = '\\'+preferred_quotation
```

Comment:
Optional: I prefer spaces around operators :+1:

Reply:
Anyway, I would definitely inline this, the variable name `add_escape` is bad :3